modules/ROOT/examples/live-demos/autocorrect/index.js

```diff
@@ -0,0 +1,10 @@
+tinymce.init({
+    selector: 'textarea#autocorrect',
+    plugins: 'autocorrect',
+    menu: {
+      autocorrect: { title: 'Autocorrect', items: 'autocorrect capitalization' }
+    },
```

Comment:
We don't normally do this and these are in the menu by default. Are we really sure we want to do this?